src/embind/embind.js

```diff
@@ -199,21 +199,11 @@ var LibraryEmbind = {
   $createNamedFunction__deps: ['$makeLegalFunctionName'],
   $createNamedFunction: function(name, body) {
     name = makeLegalFunctionName(name);
-#if DYNAMIC_EXECUTION == 0
-    return function() {
-      "use strict";
-      return body.apply(this, arguments);
-    };
-#else
-    /*jshint evil:true*/
-    return new Function(
-        "body",
-        "return function " + name + "() {\n" +
-        "    \"use strict\";" +
-        "    return body.apply(this, arguments);\n" +
-        "};\n"
-    )(body);
-#endif
+      return {
+      [name]: function () {
+        return body.apply(body, arguments);
```

Comment:
Hello, your pull request is very helpful for me.
but I don't understand why use "body.apply(body, arguments);"  in here rather than  "body.apply(this, arguments);".
In my case , I change createNamedFunction used  "body.apply(body, arguments)"  had some error.
but "body.apply(this, arguments)" is work for me.

anyway, thank you so much again,you save me to solu the CAP trouble!!